Makefile

```diff
@@ -60,6 +60,7 @@ help: default
 ### deps:             Installation dependencies
 .PHONY: deps
 deps: default
+luarocks install apisix --server https://luarocks.cn
```

Comment:
this is wrong, we can not call install apisix when call `make deps`

Reply:
Hi @membphis 

Ok, so what should I change?

cc @spacewander

Reply:
@stu01509
I have already told you **many times**.

> See my first comment. "We should wrap the server option in `make deps` ...". We need to introduce something like `LUAROCKS_SERVER=https://luarocks.cn make deps`.

https://github.com/apache/apisix/pull/3766#discussion_r589118059